etcdserver/membership/cluster.go

```diff
@@ -307,21 +307,21 @@ func (c *RaftCluster) RemoveMember(id types.ID) {
 	c.removed[id] = true
 }
 
-func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) bool {
+func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) {
```

Comment:
Well my question is why is `return Response{}` OK now when before it was sending a response `Response{s.store.Set(r.Path, r.Dir, r.Val, ttlOptions)}`-- was the `s.store.Set(...)` also `{ev: nil, err: nil}` on the old `UpdateAttributes` `return true` path? Or does it not matter?